scripts/scmigrate/migrate.go

```diff
@@ -0,0 +1,151 @@
+package scmigrate
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"sort"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/google/orderedcode"
+	dbm "github.com/tendermint/tm-db"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+type toMigrate struct {
+	key    []byte
+	commit *types.Commit
+}
+
+const prefixSeenCommit = int64(3)
+
+func makeKeyFromPrefix(ids ...interface{}) []byte {
```

Comment:
As far as I can tell, this is always called with integer values. I think we could drop some type declarations elsewhere by making this:

```suggestion
func makeKeyFromPrefix(ids ...int64) []byte {
```

Reply:
Because the type of `ordercode.Append` accepts interfaces, changing this to `int64` means we have to convert the slice, which I had been hoping to avoid, but it's fine :D